build/repo.targets

```diff
@@ -202,8 +202,8 @@
     <Error Text="Invalid configuration of %(PackageArtifact.Identity). Packages marked as AllMetapackage='true' must be Category='ship'."
            Condition="'%(PackageArtifact.Category)' != 'ship' AND '%(PackageArtifact.AllMetapackage)' == 'true' " />
 
-    <Error Text="Invalid configuration of %(PackageArtifact.Identity). Packages cannot be marked as AppMetapackage='true' and AllMetapackage='true'."
-           Condition="'%(PackageArtifact.AppMetapackage)' == 'true' AND '%(PackageArtifact.AllMetapackage)' == 'true' " />
+    <Error Text="Invalid configuration of %(PackageArtifact.Identity). Packages marked as AppMetapackage='true' must also be marked as AllMetapackage='true'."
```

Comment:
Seems like you could just automatically assume AllMetapackage == true if AppMetapackage== true

Reply:
I considered that but I thought it would be best to keep it as explicit as possible. This way, in artifacts props, it's obvious when a package goes into both .App and .All and there's no "magic" where AppMetapackages are also included in AllMetapackage.